src/api/app/assets/stylesheets/webui2/bootstrap_variables/tables.scss

```diff
@@ -1,2 +1,6 @@
 // For better contrast in tables using .table-striped
-$table-accent-bg: $gray-300;
+$table-accent-bg: $gray-100;
+
+table.dataTable tbody tr:hover {
```

Comment:
Why not [`.table-hover`](https://getbootstrap.com/docs/4.3/content/tables/#hoverable-rows)?

Reply:
> Hover won't work on mobile.

@dmarcoux I will check if there is a solution for mobile

Reply:
@hennevogel not sure what you mean exactly, there is not class ```.table-hover```?

Reply:
Instead of having this custom rule, `.table-hover` can be used.

Reply:
Ah okay, get it, okay but I will wait until we agree on having it, since i need to add this class to many tables :D